### PR TITLE
Tuik network alert

### PR DESCRIPTION
**Pull request by lundjrl**
#### Why:

GAEN-686 

#### This commit:

If we cannot complete an exposure check due to network issues, I throw a DataInaccessible error and alert the user to turn off airplane mode and turn on wifi or data.

#### How to test:

Open the app, navigate to "exposures" in the bottom tab bar, turn on airplane mode, and press "check for exposures" you should get a modal that prompts your to turn off airplane mode.

#### Linked issues:

GAEN-686 

---

### CODE REVIEW

File: src/ExposureContext.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, {
   createContext,
   useState,
   useEffect,
-  FunctionComponent,
+  FunctionComponent,
```
Comment (artgibson) [inline src/ExposureContext.tsx]: looks like some additional space here.

File: src/gaen/nativeModule.ts [hunk 2]
```diff
@@ -191,6 +192,8 @@ export const detectExposures = async (): Promise<DetectExposuresResponse> => {
         return { kind: "failure", error: "NotEnabled" }
       case "NotAuthorized":
         return { kind: "failure", error: "NotAuthorized" }
+      case "Unknown":
```
Comment (artgibson) [inline src/gaen/nativeModule.ts:195]: Can you confirm Unknown is being sent on both Android and iOS when airplane mode or networking is off? @lundjrl

**Review by artgibson — CHANGES_REQUESTED** [review]
some changes

**Comment by lundjrl** [comment]
> some changes

@artgibson Android shows "Can't check for exposures" and "Enable bluetooth" dialogs when this test is ran.

File: src/gaen/nativeModule.ts (hunk 2 above)
Comment (lundjrl) [reply src/gaen/nativeModule.ts:195]: @artgibson Android shows "Can't check for exposures" and "Enable bluetooth" dialogs when this test is ran.

Comment (artgibson) [reply src/gaen/nativeModule.ts:195]: Thanks sounds good

File: src/ExposureContext.tsx (hunk 1 above)
Comment (artgibson) [reply src/ExposureContext.tsx:5]: @lundjrl appro

**Review by artgibson — APPROVED** [review]
approved after additional space cleanup

File: ios/Gemfile.lock [hunk 3]
```diff
@@ -241,7 +241,7 @@ PLATFORMS
   x86_64-darwin-19
 
 DEPENDENCIES
-  bundler (= 2.2.1)
+  bundler (~> 2.2.1)
```
Comment (artgibson) [inline ios/Gemfile.lock:244]: Any specific reason to change this?

File: ios/Gemfile.lock [hunk 4]
```diff
@@ -251,4 +251,4 @@ RUBY VERSION
    ruby 2.6.5p114
 
 BUNDLED WITH
-   2.2.1
+   2.2.11
```
Comment (artgibson) [inline ios/Gemfile.lock:254]: Any specific reason to change this?

Comment (lundjrl) [reply ios/Gemfile.lock:254]: Absolutely not, I didn't realize it was changed... I'm thinking reinstalling some deps in my troubleshooting process did some damage

**Pull request closed without merging**